### PR TITLE
Update CODEOWNERS to include functions-framework-google

### PR DESCRIPTION
**Pull request by kenneth-rosario**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (m-strzelczyk) [inline .github/CODEOWNERS:55]: Please put the functions team as the first on the list. It's used to assign issues and PRs for review, going from left to right. The smallest group should always be on the left :)

**Review by m-strzelczyk — CHANGES_REQUESTED** [review]

**Review by dandhlee — COMMENTED** [review]
LGTM - if your team is the first team who should be taking a look at the code then could you also add your team to [`blunderbuss.yml`](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml)? If not, then fine to just update CODEOWNERS.

**Review by m-strzelczyk — APPROVED** [review]

**Comment by m-strzelczyk** [comment]
@kenneth-rosario can you reply to Dan's comment?

**Comment by kenneth-rosario** [comment]
@dandhlee My team will be mostly responsible for keeping the samples up to date.

**Review by kweinmeister — APPROVED** [review]

**Review by kweinmeister — CHANGES_REQUESTED** [review]
CODEOWNERS looks good, Kenneth!

Re: `blunderbuss.yml`, one more change which will make this seamless is to update the bug auto-assign rules here: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml#L23

Basically, you could just pull out `functions` and assign it by default to the Functions GH team you specified here.  We'll be here for additional support, this just ensures the first round of reviews is handled by the self-service team. Thanks!

**Review by nicain — APPROVED** [review]

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**
